packages/dds/merge-tree/src/test/revertibles.spec.ts

```diff
@@ -235,4 +239,107 @@ describe("MergeTree.Revertibles", () => {
 
         logger.validate();
     });
+
+    describe("Redo does not works as expected when a pair of markers and text is involved", ()=>{
+
+        generatePairwiseOptions({
+            revertMarkerInsert:[true, undefined],
```

Comment:
it wasn't 100% clear from the report, which operations we're reverted, and if they were local or ack. this pairwise matrix gives us good coverage across the possible combinations